config/cfme/manifest_5_11.json

```diff
@@ -142,6 +142,7 @@
       },
       "ems_clusters": {
         "id": null,
+	"name": null,
```

Comment:
The alignment looks to be off just a little bit

Reply:
Hm you're right I think a tab made its way in there

Reply:
Good catch, fixed